huproxyclient/client.go

```diff
@@ -89,7 +90,15 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	dialer := websocket.Dialer{}
+	dialContext := proxyplease.NewDialContext(proxyplease.Proxy{})
+
+	dialer := websocket.Dialer{
+		ReadBufferSize:   1024,
```

Comment:
Why also set these buffer sizes?

Reply:
I took it over from the example using Websockets - https://github.com/aus/proxyplease/blob/master/example/websocket.go
I don't know why they took these values, also not what values would be reasonable or if it makes sense to set them at all.
So, I could remove these settings, verify that it works for me with and without proxy, but that's it..

Reply:
Please do.